core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/store/dot-experiments-reports-store.ts

```diff
@@ -181,21 +181,41 @@ export class DotExperimentsReportsStore extends ComponentStore<DotExperimentsRep
         experimentId$.pipe(
             tap(() => this.setComponentStatus(ComponentStatus.LOADING)),
             switchMap((experimentId) =>
-                forkJoin({
-                    experiment: this.dotExperimentsService.getById(experimentId),
-                    results: this.dotExperimentsService.getResults(experimentId)
-                }).pipe(
-                    tapResponse(
-                        ({ experiment, results }) => {
-                            this.patchState({
-                                experiment: experiment,
-                                results: results
+                forkJoin([
+                    this.dotExperimentsService.getById(experimentId),
+                    this.dotExperimentsService.getResults(experimentId).pipe(
+                        catchError((response) => {
+                            const { error } = response;
+                            this.dotHttpErrorManagerService.handle({
+                                ...response,
+                                error: {
+                                    ...error,
+                                    header: error.header
+                                        ? error.header
+                                        : this.dotMessageService.get(
+                                              'dot.common.http.error.400.experiment.analytics-app-not-configured.header'
+                                          ),
+                                    message: error.message.split('.')[0]
+                                }
                             });
-                            this.updateTabTitle(experiment);
-                        },
-                        (error: HttpErrorResponse) => this.dotHttpErrorManagerService.handle(error),
-                        () => this.setComponentStatus(ComponentStatus.IDLE)
+
+                            return of(null);
+                        })
                     )
+                ]).pipe(
+                    map(([experiment, results]) => {
+                        this.patchState({
+                            experiment: experiment,
+                            results: results
+                        });
+                        this.updateTabTitle(experiment);
+                        this.setComponentStatus(ComponentStatus.IDLE);
```

Comment:
setComponentStatus and patchState together?

Reply:
Sure, added to patch all.